configuration/enabling-workspaces/en.md

```diff
@@ -30,17 +33,15 @@ Budgie provides a variety of keyboard shortcuts for workspaces.
 
 To view these shortcuts open the Keyboard application and scroll to the Navigation section.
 
-{{< altimg "workspace-shortcuts-gcc.jpg" "help-center/configuration/enabling-workspaces/" >}}
+{{< altimg "budgie-workspace-shortcuts-gcc.jpg" "help-center/configuration/enabling-workspaces/" >}}
 
 ### Workspaces Configuration
 
-There are 4 workspaces available in Budgie by default. If you would like to change this, you will need to install the
-GNOME Tweak Tool, available in the Software Center.
+There are 4 workspaces available in Budgie by default. For adding an new Workspace, you have to click on the `+` button. 
```

Comment:
`you have to` makes it sound like it's some sort of burden, should just drop that.